core/server/data/importer/importers/data/base.js

```diff
@@ -19,6 +19,19 @@ class Base {
             returnDuplicates: true
         };
 
+        this.legacyKeys = {};
+        this.legacyMapper = function legacyMapper(item) {
+            item = _.mapKeys(item, function matchLegacyKey(value, key) {
+                return self.legacyKeys[key] || key;
```

Comment:
Alright. I haven't tested this suggestion, so if you think it's better to keep a separate handling for settings, that's fine 👍